perl-lib/OESS/lib/OESS/MPLS/Discovery.pm

```diff
@@ -106,13 +106,7 @@ sub new{
 
     $self->{'db'} = OESS::Database->new(config => $config_filename);
     die if (!defined $self->{'db'});
-
-
-    $self->{'interface'} = OESS::MPLS::Discovery::Interface->new(
-        db => new OESS::DB(config => $config_filename),
-        lsp_processor => sub{ $self->lsp_handler(); }
-    );
```

Comment:
We still want this `OESS::MPLS::Discovery::Interface` object but the `lsp_processor` argument is never used; Don't pass it.